app/lib/ppom/ppom-util.test.ts

```diff
@@ -110,6 +116,27 @@ describe('PPOM Utils', () => {
       expect(spyTransactionAction).toBeCalledTimes(0);
     });
 
+    it('should not validate if request is send to users own account ', async () => {
+      const spyTransactionAction = jest.spyOn(
+        TransactionActions,
+        'setTransactionSecurityAlertResponse',
+      );
+      MockEngine.context.PreferencesController.state.securityAlertsEnabled =
+        false;
```

Comment:
```suggestion
        true;
```

I think this would be more effective if the setting were turned on which could call the validation, else we do not expect validate to run

Reply:
Its `true` for all other tests, this particular test is check that validation is not done if its `false`.

Reply:
https://github.com/MetaMask/metamask-mobile/blob/f31edcacf86dba7c35664c96afbbd9b09664cb41/app/lib/ppom/ppom-util.test.ts#L25

Reply:
in this case, I think we can remove the line then. See suggestion https://github.com/MetaMask/metamask-mobile/pull/10264/files#r1676204298

It looks like the test above checks the validation is not done when this value is false `should not validate if preference securityAlertsEnabled is false`

I think in this test, `should not validate if request is send to users own account`, we want the feature to be enabled to properly check the feature does not run for internal accounts

Reply:
I am setting value to true here: https://github.com/MetaMask/metamask-mobile/blob/f31edcacf86dba7c35664c96afbbd9b09664cb41/app/lib/ppom/ppom-util.test.ts#L25